### ISSUE TITLE
Plugin Manager does not display any plugins

### ISSUE THREAD

**Issue by jongough** [OpenCPN/plugins#13]
I have just downloaded and built the latest version (commit: ceef6d94) of OCPN and using the master branch. I have tried out the new catalog process, but I am currently not seeing any plugins being displayed. This is on my test Ubuntu system. I have tried putting the ocpn-plugins.xml from the 'data' directory into the `./opencpn` directory, but to no avail. I 'think' I got the list correctly before I tried moving to 'Experimental' but certainly since then I cannot get the list of plugins to display.

I have traced the process through and it does get the catalog and it appears to parse it correctly (status is: `catalog_status::OK`) it then goes and starts a new thread (catalog_mgr.cpp line 488) and nothing appears to happen after that. The OCPN log shows:
```
8:25:27 AEDT: Got 4 branches
18:25:27 AEDT: First branch: Alpha
18:25:42 AEDT: Effective catalog path: https://raw.githubusercontent.com/opencpn/plugins/master/ocpn-plugins.xml
```
![image](https://user-images.githubusercontent.com/2803240/71711896-1ce4a380-2e57-11ea-958a-6e9a3840da85.png)
![image](https://user-images.githubusercontent.com/2803240/71711916-266e0b80-2e57-11ea-81f2-128d077d3b3a.png)
Change to 'Experimental' and refreshed the catalog and got this in the log:
```
18:32:15 AEDT: First branch: Alpha
18:32:20 AEDT: Effective catalog path: https://raw.githubusercontent.com/opencpn/plugins/Experimental/ocpn-plugins.xml
```
and
`18:32:29 AEDT: PluginHandler: using metadata path: /home/jon/.opencpn/ocpn-plugins.xml`
Here is the ocpn-plugins.xml file:

[ocpn-plugins.xml.txt](https://github.com/OpenCPN/plugins/files/4018396/ocpn-plugins.xml.txt)

**Comment by jongough** [comment]
I have tried changing to 'Alpha' and then 'Master' and then 'Experimental' and the log now shows:
```
8:41:09 AEDT: PluginHandler: using metadata path: /home/jon/.opencpn/ocpn-plugins.xml
18:41:15 AEDT: CellDone: event 1
18:41:15 AEDT: CellDone: event 1
18:41:15 AEDT: CellDone: event 2
18:41:15 AEDT: Got 4 branches
18:41:15 AEDT: First branch: Alpha
18:41:15 AEDT: Effective catalog path: https://raw.githubusercontent.com/opencpn/plugins/Experimental/ocpn-plugins.xml
18:41:15 AEDT: CellDone: event 1
18:41:15 AEDT: CellDone: event 2
18:41:15 AEDT: Running CatalogUpdate.
```
There is still no display of plugins. Not sure if this will help

**Comment by bdbcat** [comment]
@jongough 
Try to delete your local copy of ocpn-plugins ( /home/jon/.opencpn/ocpn-plugins.xml), then update the catalog from master.

**Comment by jongough** [comment]
I have tried that: It downloads the catalog:
[ocpn-plugins.xml.txt](https://github.com/OpenCPN/plugins/files/4020858/ocpn-plugins.xml.txt)
and the log shows:
```
07:46:26 AEDT: Loading catalog for opencpn core.
07:46:26 AEDT: Getting DefaultSystemLocale...
07:46:28 AEDT: PluginHandler: using metadata path: /home/jon/.opencpn/ocpn-plugins.xml
```

**Comment by jongough** [comment]
I have found it. It is an old issue I raised to do with the name of the linux version being run on. I use linux mint, which is Ubuntu based, but the xml uses 'ubuntu' as it platform test and does not reckognise 'linuxmint' as being one and the same (from a plugin perspective). Therefore nothing is shown to the user. If I hand change the xml file to be <target>linuxmint</target> all the plugins show up.
